idaplugin/rematch/network.py

```diff
@@ -12,8 +12,12 @@
 cookiejar = CookieJar()
 opener = urllib2.build_opener(urllib2.HTTPCookieProcessor(cookiejar))
 
-_threadpool = QtCore.QThreadPool()
-_threadpool.setMaxThreadCount(config['network']['threadcount'])
+try:
+  _threadpool = QtCore.QThreadPool()
+  _threadpool.setMaxThreadCount(config['network']['threadcount'])
+except KeyError as err:
+ from . import config
+ config = config.Config()
```

Comment:
If you want to log this warning I think it better conform to how we log stuff in other places, using the logger API. However this doesn't seem like a major error and an hard-coded default value seems like a simpler approach. Up to you.

also IIRC `traceback.print_exc()` and `traceback.print_exception(*sys.exc_info())` do the same thing.